modelzoo/dien/README.md

```diff
@@ -13,18 +13,33 @@ export path_to_e2eaiok=`pwd`/e2eAIOK
 mkdir -p ${path_to_e2eaiok_dataset}
 ```
 
-## Install
+## Install End to end AIOK Kit for DIEN
```

Comment:
should use full name:  Intel® End-to-End AI Optimization Kit.

Reply:
@Jian-Zhang I fixed naming issue